organize_content/exercise2-organize-content.md

```diff
@@ -3,21 +3,27 @@
 Git is a distributed version control system, which means you can work locally,
 then share or *push* your changes to a server. In this case, the server you push to is GitLab.
 
+GitLab uses the SSH protocol to securely communicate with Git.
+When you use SSH keys to authenticate to the GitLab remote server,
+you don't need to supply your username and password each time.
+
+To use SSH to communicate with GitLab, you need:
+
+- The OpenSSH client, which comes pre-installed on GNU/Linux, macOS, and Windows 10.
+- SSH version 6.5 or later. Earlier versions used an MD5 signature, which is not secure.
+- To view the version of SSH installed on your system, run `ssh -V`.
 SSH uses two keys, a public key and a private key.
 
 - The public key can be distributed.
 - The private key should be protected.
 
-GitLab uses the SSH protocol to securely communicate with Git.
-When you use SSH keys to authenticate to the GitLab remote server,
-you don't need to supply your username and password each time.
-
 Make sure your private key has the right permissions, otherwise it could be stolen.
```

Comment:
**Comment** @Preshh0 The private key doesn't yet exist, so I suggest moving this to after the private key has been created.